packages/client/src/client.js

```diff
@@ -516,6 +516,21 @@ export class PercyClient {
     return comparison;
   }
 
+  async sendFailedEvents(buildId, { errorKind = 'sdk', client = null, clientVersion = null, cliVersion = null, errorMessage = null } = {}) {
+    validateId('build', buildId);
+    this.log.debug('Sending FailedEvents');
+    return this.post(`builds/${buildId}/failed-events`, {
+      data: {
+        buildId: buildId,
```

Comment:
why do we need this if its already sent in endpoint path ?

Reply:
it was added because in api pr it was missing, removed and added build_id in api pr.